pom.xml

```diff
@@ -124,6 +124,16 @@
                 <artifactId>gson</artifactId>
                 <version>${version.com.google.code.gson}</version>
             </dependency>
+            <dependency>
+                <groupId>com.fasterxml.uuid</groupId>
```

Comment:
Shall we remove this as this is no longer needed?

Reply:
fixed with 85f0cb99efc99bb31ecb52a36a7f647e497f4dc4